src/libraries/Native/Unix/System.Security.Cryptography.Native/pal_ssl.c

```diff
@@ -16,6 +16,16 @@ c_static_assert(PAL_SSL_ERROR_WANT_WRITE == SSL_ERROR_WANT_WRITE);
 c_static_assert(PAL_SSL_ERROR_SYSCALL == SSL_ERROR_SYSCALL);
 c_static_assert(PAL_SSL_ERROR_ZERO_RETURN == SSL_ERROR_ZERO_RETURN);
 
+#define DOTNET_DEFAULT_CIPHERSTRING \
```

Comment:
Do you need TLS 1.3 cipher suite names in here like `TLS_AES_256_GCM_SHA384`? These cipher suites below look like they only apply to TLS 1.2 and below.

Reply:
It's correct that TLS 1.3 isn't included here, because it's a different API in OpenSSL.  (In config they're CipherString (SSL2-TLS1.2) and CipherSuites (TLS 1.3))

Reply:
Ah, I see further down below, thanks.